bionic/util.py

```diff
@@ -312,6 +313,47 @@ def exception(self, msg, *args, **kwargs):
             logging.ERROR, msg, *args, exc_info=True, **kwargs)
 
 
+class FileCopier(object):
+    """
+    A wrapper for a Path object
+
+
+    Parameters
+    ----------
+    name: String
+        The name of the computed entity.
+
+    src_file_path: Path
+        A path to the persisted file.
+    """
+    def __init__(self, name, src_file_path):
+        self.name = name
+        self.src_file_path = src_file_path
+
+    def copy(self, destination):
+        """
+        Copies file at `self.src_file_path` to `destination`
```

Comment:
nit: The reader of these docs probably doesn't care about `self.src_file_path`.  Depending on what we write for the class-level documentation, there's probably a clearer way to express this.

Reply:
Updating class-level docstring to be `A wrapper for a Path object, exposing a copy method that will copy the underlying file to a local or cloud destination`, so will likely change this comment to avoid referring to `self.src_file_path`.  Lmk if you disagree tho!

Reply:
Works for me!  Only suggestion I would have (*) would be maybe `file path` instead of `path object`?

(*) Not counting the previous 41 suggestions, obv.